lib/web_ui/test/text/font_collection_test.dart

```diff
@@ -165,9 +165,9 @@ void testMain() {
         await fontManager.downloadAllFonts();
         fontManager.registerDownloadedFonts();
         domDocument.fonts!
-            .forEach(allowInterop((DomFontFace f, DomFontFace f2, DomFontFaceSet s) {
+            .forEach((DomFontFace f, DomFontFace f2, DomFontFaceSet s) {
```

Comment:
Woops, the large diff didn't load for me. Ignore.

Reply:
Hah yea, sorry. I moved the `toJS` internal to the shim.

The really wonderful super nice thing about the direction we are going is that soon we won't even have to ask questions like this because Dart functions won't even be able to flow to JS unless someone calls `toJS` *somewhere* in the codebase. First we have to get to strict mode...

JS types++